static/app/utils/performance/quickTrace/utils.tsx

```diff
@@ -102,7 +101,8 @@ export function flattenRelevantPaths(
 }
 
 function simplifyEvent(event: TraceFull): QuickTraceEvent {
```

Comment:
I think we should just delete this function

Reply:
Agree, I'm just trying to keep the number of changes down to a min in this PR